draft-ietf-quic-recovery.md

```diff
@@ -417,9 +417,9 @@ to accelerate loss recovery.  The receiver SHOULD send an immediate ACK
 when it receives a new packet which is not one greater than the
 largest received packet number.
 
-Also, reception of an packet marked as ECN Congestion Experience
+Similarly, reception of an packet marked as ECN Congestion Experience
 (ECN-CE) SHOULD be acknowledged immediately to quicker react to
-congesiton events. Additional ECN-CE marks received during the same
+congestion events. Additional ECN-CE marks received during the same
```

Comment:
I don't think you need to say "during the same recovery period" since we're not proscribing any receiver side behavior corresponding to whether the marks are received within the same recovery period.